payments/stripe/__init__.py

```diff
@@ -1,14 +1,21 @@
 import json
+import warnings
+from dataclasses import asdict, dataclass, field
 from decimal import Decimal
+from typing import Optional
 
-import stripe
+from django.utils.translation import gettext as _
 
-from .. import PaymentError
-from .. import PaymentStatus
-from .. import RedirectNeeded
+from .. import PaymentError, PaymentStatus, RedirectNeeded
 from ..core import BasicProvider
-from .forms import ModalPaymentForm
-from .forms import PaymentForm
+from .forms import ModalPaymentForm, PaymentForm, PaymentFormV3
+
+try:
+    import stripe
+except ImportError as exc:
+    raise ImportError(
+        "You need to install `stripe>=5.4.0` onto your environment"
```

Comment:
I'd rather not specify the version. We'll update `setup.py` at some point and this will fall out of sync.